iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/telemetry/ReceiveMessagesTests.java

```diff
@@ -10,12 +10,13 @@
 import com.microsoft.azure.sdk.iot.device.DeviceClient;
 import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
 import com.microsoft.azure.sdk.iot.device.ModuleClient;
-import com.microsoft.azure.sdk.iot.service.Module;
+import com.microsoft.azure.sdk.iot.service.*;
 import com.microsoft.azure.sdk.iot.service.auth.AuthenticationType;
```

Comment:
if line 13 is *, is this explicit import no longer needed?

Reply:
Disregard these changes. There was no reason for me to change this file after all, so I've reverted my changes here